config/core/deployments/webhook.yaml

```diff
@@ -60,8 +60,14 @@ spec:
           value: knative.dev/eventing
         - name: WEBHOOK_NAME
           value: eventing-webhook
-        - name: SINK_BINDING_OPT_OUT_SELECTOR
-          value: "false"
+          # SINK_BINDING_SELECTOR can be either inclusion or exclusion (case insensitive).
```

Comment:
it looks pretty case sensitive to me.

Reply:
Indeed! I was making an update to that statement...

Reply:
Updated document. it should be all lower case.